src/app/onion/learning-object-builder/builder-store.service.ts

```diff
@@ -1053,7 +1060,13 @@ export class BuilderStore {
       .then(() => {
         this.serviceInteraction$.next(false);
       })
-      .catch(e => this.handleServiceError(e, BUILDER_ERRORS.UPDATE_OUTCOME));
+      .catch(e => {
+        if (e.status !== 406) {
+          this.handleServiceError(e, BUILDER_ERRORS.UPDATE_OUTCOME)
```

Comment:
Couldn't we add the 406 to this function?

Reply:
I agree